cloudferrylib/utils/qemu_img.py

```diff
@@ -51,7 +51,13 @@ def convert_image(self,
 
     def detect_backing_file(self, dest_disk_ephemeral, host_instance):
         cmd = self.backing_file_cmd(dest_disk_ephemeral)
-        return self.parsing_output_backing(self.execute(cmd, host_instance))
+        try:
+            ret = self.parsing_output_backing(self.execute(cmd, host_instance))
+        except Exception as e:
+            #TODO need to find good solution when when there is no backing file
```

Comment:
I'm assuming this is still work in progress, but anyway leaving comment so that it does not get merged:
Let's implement this TODO as part of the patch: catching `Exception` is considered bad practice. Also `print` should be replaced with `LOG`